features/facebookImport/src/components/activitiesMinistory/activitiesMinistory.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+
+import i18n from "../../i18n";
+
+import BarChartHorizontal from "../dataViz/barChartHorizontal.jsx";
+
+import { fillArray } from "../dataViz/utils.jsx";
+
+import "./activitiesMinistory.css";
+import "./datePicker.css";
+import "../tabs/tabs.css";
+
+const monthsAbbreviation = i18n.t("common:months.abbreviation").split(" ");
```

Comment:
This looks like something we should have at the core level :thinking:

Reply:
Something which would go to `poly-look/src/incubate` i'd suppose as this is pretty much only used in graphs right?